lib/rucio/core/rule.py

```diff
@@ -1850,7 +1850,7 @@ def update_rules_for_bad_replica(scope, name, rse_id, nowait=False, session=None
     if nlock:
         session.query(models.RSEFileAssociation).filter(models.RSEFileAssociation.scope == scope, models.RSEFileAssociation.name == name, models.RSEFileAssociation.rse_id == rse_id).update({'state': ReplicaState.COPYING})
     else:
-        logging.info('File %s:%s at site %s has no locks. Will be deleted now.', scope, name, rse)
+        logging.info('File %s:%s at site %s has no locks. Will be deleted now.', scope, name, rse_id)
```

Comment:
for all the logging it would be good to have both RSE name and RSE ID

Reply:
Good point, I can update it to a format like `logging.info('File %s:%s at site %s(%s) has no locks. Will be deleted now.', scope, name, get_rse_name(rse_id=rse_id, session=session), rse_id) ` throught core

Reply:
We could even leave rse name for logging. I mean, in the case of multi-VO, the VO-Name would be encoded in the scope anyway, so it is somewhat clear which RSE is concerned.

Reply:
If we're using the external representation for rse, it'd be cleaner to use the external representation for all scope and accounts too and explicitly stating the vo. In which case just the name is fine here and when VO is added, it will need to be inserted into all the logging.

Reply:
I've updated the core logging to use rse names instead of ids.